docs/build/tutorial.rst

```diff
@@ -200,6 +200,12 @@ The file generated with the "generic" configuration looks like::
     # black.entrypoint = black
     # black.options = -l 79 REVISION_SCRIPT_FILENAME
 
+    # lint with attempts to fix using "ruff" - use the exec runner, execute a binary
```

Comment:
**mike bayer** (zzzeek) wrote:

Done

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/alembic/+/4788